addons/account/views/account_move_views.xml

```diff
@@ -423,20 +422,23 @@
                         ('name', 'ilike', self), ('ref', 'ilike', self), ('account_id', 'ilike', self), ('partner_id', 'ilike', self)]"/>
                     <field name="name"/>
                     <field name="ref"/>
-                    <field name="payment_date" string="Due Date"/>
+                    <field name="payment_date" string="Next Payment Date"/>
                     <field name="partner_id"/>
                     <field name="journal_id"/>
                     <field name="currency_id" groups="base.group_multi_currency" invisible="1"/>
                     <field name="company_currency_id" invisible="1"/>
                     <separator/>
                     <filter string="Posted" name="posted" domain="[('parent_state', '=', 'posted')]" help="Posted Journal Items" invisible="1"/>
                     <separator/>
-                    <filter string="Sales" name="sales" domain="[('journal_id.type', '=', 'sale')]" context="{'default_journal_type': 'sale'}"/>
```

Comment:
Have you checked that this does not break anything ? I see other parts of the code that use this filter

Reply:
Up

Reply:
The filter "Sales" you see being used in other places is the filter of account.move.line.search, not this one which is/was in account.move.line.payment.search. Nothing is broken when removing this one.